docs/design/storage-node-logger.md

```diff
@@ -0,0 +1,30 @@
+# Rotating Logger for Storage Node
+
+## Abstract
+
+Storage Node needs a logger that can log to disk, rotate files, compress old files, delete really old files, and ensure a limit on the size of logs.
+
+## Background
+
+Currently log files are agglomerating and are completely unmanaged. Storage nodes need to be able to keep the log files under control to prevent logs from using an exorbitant amount of the storage node's hard drive space.
+
+## Design
+
+Logger Service that runs at storage node startup
+* Write files to disk when log function is called.
+* Rotate the current file being used for logging by current date and log file size.
+* Compress log files older than 3(?) days.
```

Comment:
We can immediately compress them after rotating to a new day at least initially, since the separate compression logic would require more code.